yarn.lock

```diff
@@ -4205,3 +4292,7 @@ yargs@~3.10.0:
     cliui "^2.1.0"
     decamelize "^1.0.0"
     window-size "0.1.0"
+
+yarn@^1.3.2:
```

Comment:
Not sure why there's a dev dependency on `yarn`?

Reply:
Ther lerna setup uses `yarn` as npmClient. I thought it would be wise to specify that dependency explicitly instead of assuming there was some globally installed version present

Reply:
We have a yarn.lock, this means developers should be using yarn, not npm, to install dependencies. I think it's safe to assume this, just as we didn't have a local dependency on npm when we used that to install packages.

Reply:
Ok, will revert